andela/team-odd-bn-backend
# #169258618  Sequelize setup

#### What does this PR do?
- This PR sets up sequelize

#### Description of Task to be completed
- [X] Install sequelize `npm install sequelize sequelize-cli`
- [X] Install pg and pg h-store `npm install pg pg-hstore`
- [x] Update the package.json file.
- [X] Initialize sequelize `npm db-init`
- [X] Create a dummy`npm dummy-model`
- [X] Do migrations `npm db-migrate`
- [x] Write the descriptions in the README.md

#### How should this be manually tested?
- git clone `https://github.com/andela/team-odd-bn-backend.git`
- git checkout `ch-setup-sequelize-169258618`
- Fill the .env file with your details in the format in tthe .env.example file
- Do `npm install` to install dependencies.
- Do `npm run db-migrate` to make migrations.
- Go to the psql cli to confirm that  the table have been created 

#### Any background context you want to provide?
 I have already created a dummy model, However if you want to create another model of your choice =, go to package.json file in the scripts find "dummy model", you can edit that script or add another script.
#### What are the relevant pivotal tracker stories?
- [#169258618](https://www.pivotaltracker.com/n/projects/2407428/stories/169258618)
#### Screenshots (if appropriate)
<img width="445" alt="Screen Shot 2019-11-05 at 09 56 43" src="https://user-images.githubusercontent.com/45129725/68188807-a0e2f200-ffb2-11e9-956d-d55cb3f7cc32.png">
